pkg/tsdb/prometheus/prometeus_bench_test.go

```diff
@@ -1,24 +1,48 @@
 package prometheus
```

Comment:
I moved this into it's own file so it's clear. Here we can distinguish it from other prometheus tests using `bench_test.go`. I've also seen things like `integration_test.go` etc.